src/main/scala/io/iohk/ethereum/vm/WorldStateProxy.scala

```diff
@@ -29,6 +29,7 @@ trait WorldStateProxy[WS <: WorldStateProxy[WS, S], S <: Storage[S]] {
 
   def getCode(address: Address): ByteString
   def getStorage(address: Address): S
+  def getBlockHash(number: BigInt): Option[ByteString]
```

Comment:
@rtkaczyk yes....I think it will, actually I have the block info and i'm not using it within the WorldState